source/crud/tests/unified/bulkWrite-updateMany-let.yml

```diff
@@ -0,0 +1,107 @@
+description: "BulkWrite updateMany-let"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1, x: 20 }
+      - { _id: 2, x: 21 }
+
+tests:
+  - description: "BulkWrite updateMany with let option"
+    runOnRequirements:
+      - minServerVersion: "5.0"
+    operations:
+      - object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - updateMany:
+                filter: &filter
+                  $expr:
+                    $eq: [ "$_id", "$$id" ]
+                update: &update
```

Comment:
This test likely requires a pipeline-style update parameter. See the corresponding test in [updateOne-let.yml](https://github.com/mongodb/specifications/blob/master/source/crud/tests/unified/updateOne-let.yml) and [DRIVERS-2210](https://jira.mongodb.org/browse/DRIVERS-2210) for context.

Likewise `bulkWrite-updateOne-let.yml`.

Reply:
Fixed.